escuela-nacional-de-antropologia-e-historia-author-date.csl

```diff
@@ -486,15 +486,13 @@
     </layout>
   </citation>
   <!-- Para generar la bibliografía -->
-  <bibliography et-al-min="4" et-al-use-first="3" subsequent-author-substitute="" entry-spacing="0">
+  <bibliography et-al-min="4" et-al-use-first="3" subsequent-author-substitute="" entry-spacing="0" line-spacing="1">
     <sort>
       <key macro="contributors"/>
       <key macro="date"/>
     </sort>
     <layout suffix=".">
-      <group display="block">
-        <text macro="contributors"/>
-      </group>
+      <text macro="contributors" suffix='&#xD;'/>
```

Comment:
@adam3smith, is a hard return acceptable? I guess it would typically result in less spacing than `display="block"` with HTML styling, which appears to be the goal here.